### PR TITLE
wrote tests for signer universal package

### PR DESCRIPTION
**Pull request by timestruck**

### CODE REVIEW

File: signer/universal/universal_test.go [hunk 1]
```diff
@@ -1 +1,21 @@
...
+	}
+	var r = &Root{
+		Config:      h,
+		ForceRemote: false,
+	}
+	_, _, err := pkcs11Signer(r, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
```
Comment (kisom) [inline signer/universal/universal_test.go]: Extraneous newline.

File: signer/universal/universal_test.go [hunk 2]
```diff
@@ -1 +1,41 @@
 package universal
+
+import (
+	"github.com/cloudflare/cfssl/config"
+	"testing"
+	"time"
+)
+
+var expiry = 1 * time.Minute
+var validConfig = &config.Config{
```
Comment (lziest) [inline signer/universal/universal_test.go]: call it validLocalConfig

File: signer/universal/universal_test.go [hunk 3]
```diff
@@ -1 +1,41 @@
...
+			},
+		},
+		Default: &config.SigningProfile{
+			Usage:  []string{"digital signature"},
+			Expiry: expiry,
+		},
+	},
+}
+
+func TestNewPKCS11Signer(t *testing.T) {
```
Comment (lziest) [inline signer/universal/universal_test.go]: change the name :) e.g. TestNewSigner

File: signer/universal/universal_test.go [hunk 4]
```diff
@@ -1 +1,41 @@
 package universal
+
+import (
+	"github.com/cloudflare/cfssl/config"
```
Comment (lziest) [inline signer/universal/universal_test.go]: style nit: this should be put below standard libarary and separated with one blank line.

**Comment by lziest** [comment]
Other than those style and wording issues, looks good to me.

**Comment by lziest** [comment]
LGTM, please rebase and squash the changes into one commit. Thanks.

File: signer/universal/universal_test.go [hunk 5]
```diff
@@ -1 +1,42 @@
...
+		},
+		Default: &config.SigningProfile{
+			Usage:  []string{"digital signature"},
+			Expiry: expiry,
+		},
+	},
+}
+
+func TestNewSigner(t *testing.T) {
+
```
Comment (lziest) [inline signer/universal/universal_test.go]: erase this blank line.

File: signer/universal/universal_test.go [hunk 6]
```diff
@@ -1 +1,42 @@
...
+	h := map[string]string{
+		"key-file":  "../local/testdata/ca_key.pem",
+		"cert-file": "../local/testdata/ca.pem",
+	}
+
+	r := &Root{
+		Config:      h,
+		ForceRemote: false,
+	}
+	_, err := NewSigner(*r, validLocalConfig.Signing)
```
Comment (lziest) [inline signer/universal/universal_test.go]: add a blank line here.

**Comment by lziest** [comment]
Finally, after squashing the commit into one, you should rewrite the commit message, i.e. erase the two sub-commit messages. I think "Add a test for signer/universal package" should be good.

**Comment by lziest** [comment]
Thanks for your contribution.

**Comment by lziest** [comment]
LGTM

**Pull request merged**